src/app/containers/ChartbeatAnalytics/utils/index.test.js

```diff
@@ -784,6 +790,86 @@ describe('Chartbeat utilities', () => {
     expect(getConfig(fixtureData)).toStrictEqual(expectedConfig);
   });
 
+  it('should return config for canonical pages when page type is FIX and env is not live', () => {
+    const fixtureData = {
+      isAmp: false,
+      platform: 'canonical',
+      pageType: 'FIX',
+      data: {},
+      brandName: 'BBC-Afique',
+      chartbeatDomain: 'bbc.co.uk',
+      env: 'test',
+      service: 'afrique',
+      origin: 'test.bbc.com',
+      previousPath: '/previous-path',
+    };
+
+    const expectedConfig = {
+      domain: 'test.bbc.co.uk',
+      idSync: {
+        bbc_hid: 'foobar',
+      },
+      path: '/',
+      sections: 'Afrique, Afrique - FIX',
+      title: 'This is a Feautre Index page title',
```

Comment:
Typo? `This is a Feature Index page title`

Reply:
DOH